go/runtime/scheduling/simple/txpool/orderedmap/ordered_map.go

```diff
@@ -188,12 +188,13 @@ func (q *orderedMap) UpdateConfig(cfg api.Config) error {
 		if current == nil {
 			break
 		}
-		if cfg.MaxPoolSize > uint64(newQueue.Len()) {
+		if uint64(newQueue.Len()) >= cfg.MaxPoolSize {
 			break
 		}
 		el := current.Value.(*pair)
 		txSize := uint64(len(el.Value))
 		if txSize > cfg.MaxBatchSizeBytes {
+			current = current.Prev()
```

Comment:
:facepalm: 
Can you please also add a simple unit test for update configuration to `scheduling/simple/txpool/tester.go`

Reply:
Oh I see, there is one for the scheduler (this is the one that caught this) but not for the txpool. Will add it here as well.

Reply:
Done.